apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -1740,8 +1739,7 @@ def add_matrix_to_collection(
     context: Optional[SOMATileDBContext] = None,
 ) -> None:
     """This is useful for adding X/obsp/varm/etc data, for example from
-    `Scanpy <https://scanpy.readthedocs.io/>`_'s ``scanpy.pp.normalize_total``,
-    ``scanpy.pp.log1p``, etc.
+    Scanpy's ``scanpy.pp.normalize_total``, ``scanpy.pp.log1p``, etc.
```

Comment:
I really love the idea of linking to another package. Sadly, in the first-sentence-only summaries that Sphinx loves to create, it somehow can't handle these and leaves the summary _entirely blank_. Unacceptable.